src/data_storage/import/data_staging_storage.ts

```diff
@@ -63,9 +58,20 @@ export default class DataStagingStorage extends PostgresStorage {
     }
 
     public async List(importID: string, offset:number, limit:number): Promise<Result<DataStagingT[]>>{
+        if(limit === -1) { // allow return all with a negative number
```

Comment:
Is this behavior that we have used anywhere else or is it unique? If it's unique, are there other patterns we have followed elsewhere?

Reply:
it's being used somewhere else now. I should update the rest of the storage layers with it.